features/0023-did-exchange/README.md

```diff
@@ -434,4 +434,4 @@ The following lists the implementations (if any) of this RFC. Please do a pull r
 
 Name / Link | Implementation Notes
 --- | ---
- |  |
+[Aries Protocol Test Suite](https://github.com/hyperledger/aries-protocol-test-suite) | [test_manual.py](https://github.com/hyperledger/aries-protocol-test-suite/blob/master/protocol_tests/connection/test_manual.py)
```

Comment:
The protocol test suite currently tests the `connection` protocol, not the `did-exchange` one. Also, it seems to be testing an older version that doesn't use the signature decorator.

Reply:
Yes, I believe that has changed since I raised this PR; @dbluhm updated it for IIW. So should we take this impl line out?

Reply:
For sure it isn't testing for `did-exchange`.

But I was wrong about it not testing for the signature decorator - it, in fact, is: 

https://github.com/hyperledger/aries-protocol-test-suite/blob/15036428d41de75fa55e8b09b085bd25aa01a280/protocol_tests/connection/test_manual.py#L55-L63

Reply:
I suggest we remove this line from did-exchange and move it over to the `connection` RFC.

Reply:
@dhh1128 you missed this one